package.json

```diff
@@ -32,7 +32,6 @@
     "react-modal": "3.16.1",
     "react-paginating": "1.4.0",
     "react-protected-mailto": "1.0.3",
-    "react-router-bootstrap": "0.25.0",
```

Comment:
We don't seem to use this anywhere, and there are not occurrences of `LinkContainer` in the codebase. I `git blame`d the history and I saw it was pulled in when the project was converted to react.

Reply:
When I find these cases, I put in a PR to remove the unused dependency.